fe/fe-core/src/main/java/com/starrocks/catalog/TableProperty.java

```diff
@@ -559,7 +559,7 @@ public static String persistentIndexTypeToString(TPersistentIndexType type) {
     public TableProperty buildConstraint() {
         if (properties.containsKey(PropertyAnalyzer.PROPERTIES_UNIQUE_CONSTRAINT)) {
             try {
-                uniqueConstraints = UniqueConstraint.parse(
+                uniqueConstraints = UniqueConstraint.parse(null, null, null,
```

Comment:
I think you could give default catalogName and db name, not null

Reply:
It's hard to get these information at this place. and these properties will be parsed from the uk fk constraint statements.